packages/plans-grid/src/plans-grid/style.scss

```diff
@@ -20,6 +20,14 @@
 	align-items: center;
 }
 
+.plans-grid__toggle {
+	margin: -24px 0 32px;
+
+	@include break-medium {
+		margin-bottom: 40px;
+	}
+}
```

Comment:
Are we planning to reuse the `PlansIntervalToggle`? If that's the case, it's better to add the margins to the `.plans-grid__table`. That prevents the component from being too opinionated.

Reply:
I thought this was cleaner — in case we ever remove the toggle, you'd only need to delete the toggle styles.

Also, the toggle has a negative margin top — which means that the alternative would be to change the margin bottom of `.plans-grid__header`. But that margin comes from the `onboarding-heading-padding` mixin, which is reused across a lot of other components. So I decided to leave that untouched, and use a negative margin on the toggle instead.

Reply:
It makes total sense to reuse as much as possible so it makes sense to do that here as well. Thanks for explaining 👍